app.js

```diff
@@ -1 +1,32 @@
+'use strict';
+//var firstPikeMain ={
+var hours = ['6a','7a','8a','9a','10a','11a','12p','1p','2p','3p','4p','5p','6p','7p','8p'];
+var firstPike = {
+  minCust: 23, //object properties //for mathrandom
+  maxCust: 65, //object ptoperties//for mathrandom
+};
 
+var avgCust = function() {
+  var a = Math.floor(Math.random() * (firstPike.maxCust - firstPike.minCust)) + firstPike.minCust;
+  return a;
+};
+var hourCust = [];//avgCust(); calls funtion
+for (var j = 0; j < hours.length; j++){
+  //code
+  var random = avgCust();//random stores avgCust output values
+  hourCust.push(random);//hourCust creates the array from values stored in random
+};
```

Comment:
This loop should be stuck in a function. There shouldn't be any loops at the global level